### PR TITLE
NodeJS, Express 102: Clarify models description

### PR DESCRIPTION
**Pull request by OmarNaguib**
## Because

original text seems to suggest that you have to make a model for each "entry", which can be understood as a single object/document

## This PR

-clarifies what a model represents

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project Contributing Guide](https://github.com/TheOdinProject/.github/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: nodeJS/express_and_mongoose/express_102_crud_and_mvc.md [hunk 1]
```diff
@@ -30,7 +30,7 @@ The CRUD operations roughly correlate to the HTTP methods that you can employ in
 
 MVC is another common concept in web development and also something that is likely to come up in an interview question.  MVC stands for _Model, View, Controller_ and refers to the architecture of your code. Basically, it is a way to organize your application by separating all of the actions into 3 main components: Models, Views and Controllers.
 
-<span id="model">**Models**</span> are the basic building blocks of your database. So for every entry in your DB (books, authors, etc. in our Library Project), you'll create a model that holds the details of that entry. Models define the types of information that get used by your views and controllers.
+<span id="model">**Models**</span> are the basic building blocks of your database. So for every type of entries in your DB (books, authors, etc. in our Library Project), you'll create a model that holds the details of that entry. Models define the types of information that get used by your views and controllers.
```
Comment (rlmoser99) [inline nodeJS/express_and_mongoose/express_102_crud_and_mvc.md:33]: I am stuck on "entries" being plural and it doesn't quite sound right to me.  What do you think of using "type of entry"?

**Review by rlmoser99 — CHANGES_REQUESTED** [review]
Great catch! Thank you for taking the time to make a PR to address this possible confusion. I have one suggestion, but I am open to your opinion.

File: nodeJS/express_and_mongoose/express_102_crud_and_mvc.md (hunk 1 above)
Comment (OmarNaguib) [reply nodeJS/express_and_mongoose/express_102_crud_and_mvc.md:33]: I thought about this too 😅 
I appears both are correct
image
whatever sounds better for you,
 if we change it to be singular might we also change `(books, authors, etc. in our Library Project)` to (book, author, etc. in our Library Project)? (it's why I ended up choosing plural)

Comment (OmarNaguib) [reply nodeJS/express_and_mongoose/express_102_crud_and_mvc.md]: I noticed this sentence `you'll create a model that holds the details of that entry` should "entry" here be changed to "type" or "type of entry?

Comment (rlmoser99) [reply nodeJS/express_and_mongoose/express_102_crud_and_mvc.md]: Good catch. That sentence does need some updating. I am open to suggestions, but in addition to adding "type of entry" to it, I believe "will hold" reads better then just "holds"

> Models are the basic building blocks of your database. So for every type of entry in your DB (book, author, etc. in our Library Project), you'll create a model that will hold the details of that type of entry. Models define the types of information that get used by your views and controllers.

Comment (OmarNaguib) [reply nodeJS/express_and_mongoose/express_102_crud_and_mvc.md]: agreed!

File: nodeJS/express_and_mongoose/express_102_crud_and_mvc.md [hunk 2]
```diff
@@ -30,7 +30,7 @@ The CRUD operations roughly correlate to the HTTP methods that you can employ in
 
 MVC is another common concept in web development and also something that is likely to come up in an interview question.  MVC stands for _Model, View, Controller_ and refers to the architecture of your code. Basically, it is a way to organize your application by separating all of the actions into 3 main components: Models, Views and Controllers.
 
-<span id="model">**Models**</span> are the basic building blocks of your database. So for every entry in your DB (books, authors, etc. in our Library Project), you'll create a model that holds the details of that entry. Models define the types of information that get used by your views and controllers.
+<span id="model">**Models**</span> are the basic building blocks of your database. So for every type of entry in your DB (book, author, etc. in our Library Project), you'll create a model that holds the details of that entry. Models define the types of information that get used by your views and controllers.
```
Comment (rlmoser99) [inline nodeJS/express_and_mongoose/express_102_crud_and_mvc.md:33]: Just to make sure we are on the same page. This is what I believe we've agreed on:

```suggestion
<span id="model">**Models**</span> are the basic building blocks of your database. So for every type of entry in your DB (book, author, etc. in our Library Project), you'll create a model that will hold the details of that type of entry. Models define the types of information that get used by your views and controllers.
```

Comment (OmarNaguib) [reply nodeJS/express_and_mongoose/express_102_crud_and_mvc.md:33]: That's perfect

**Review by rlmoser99 — APPROVED** [review]
Thanks so much for taking the time to point out this possible confusing wording and for helping make sure the entire paragraph made sense with these changes!

**Comment by OmarNaguib** [comment]
And thank you for all your help and for your wholesome interactions.

**Pull request merged**